webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -10161,7 +10168,18 @@ var userOverride = {
                     contact.contactId,
                     isGroupTab
                 );
-                var imgsrctag = _this.getContactImageLink(contact, displayName);
+                var imgsrctag = '';
+                var isConversationInWaitingQueue = false
+                if (contact && contact.metadata && contact.metadata.CONVERSATION_STATUS == 7 ){
+                    // if conversation is in waiting queue
+                    displayName = MCK_LABELS['waiting.queue.message']['contact.name'];
+                    imgsrctag = '<div class="mck-videocall-image alpha_W"><span class="mck-contact-icon">...</span></div>';
```

Comment:
what are these classes `mck-videocall-image alpha_W `

Reply:
I am not aware of these classes. 
`imgsrctag = '<div class="mck-videocall-image alpha_W"><span class="mck-contact-icon">...</span></div>';`
Originally, the above template was having 'W' instead of '...'